### ISSUE TITLE
Malware, Doesnt work + You nuked my logs before

### ISSUE THREAD

**Issue by CoolEpikSwagger** [moom825/Discord-RAT-2.0#23]
Nuked my logs channel before, You aint funny smh

**Comment by moom825** [comment]
you goofen?

**Comment by moom825** [comment]
why and how would i do that?

**Comment by moom825** [comment]
oh your one of the people who post malware on github, yea my scanner picks that up....maybe dont do it....

**Comment by CoolEpikSwagger** [comment]
Don't act like you're innocent lol
![image](https://user-images.githubusercontent.com/120762934/211123526-5d1a7e11-324e-434f-bb3e-844f34c53a56.png)

**Comment by moom825** [comment]
huh? that aint me?

**Comment by moom825** [comment]
i either confront people on an account called "why", which has been renamed to be my alt (now named "moom825") or on my main which is also "moom825".

**Comment by moom825** [comment]
also, you know your bot token is embeded into the exe if send to people right? anyone you send it too has access to the bot token...

**Comment by CoolEpikSwagger** [comment]
Yeah but the funny part is I never gave out the exe, So there is no other way they could have gotten their hands on it.

**Comment by moom825** [comment]
yea, well funny thing is, you can open whatever you used to build the exe in a tool like dnspy or ilspy and veiw the source code directly. so how about you do that before making shit up

**Comment by moom825** [comment]
and then you can go ahead tell me where exactly the backdoor is, and how im "logging people", because ill tell, i dont care enough to do that

**Comment by CoolEpikSwagger** [comment]
> yea, well funny thing is, you can open whatever you used to build the exe in a tool like dnspy or ilspy and veiw the source code directly. so how about you do that before making shit up

Elaborate on how I'm making shit up

**Comment by moom825** [comment]
because i dont create logs whenever someone builds something. nor do i recieve a bot token of anykind

**Comment by sefinek24** [comment]
https://user-images.githubusercontent.com/92880265/211455345-d54b6a0c-15b6-43b4-9aac-e2d5eadb7b75.mov

**Comment by CoolEpikSwagger** [comment]
> because i dont create logs whenever someone builds something. nor do i recieve a bot token of anykind

Sure bro then how else did they get the token, and just so happen to link your github whilst nuking the log server

**Comment by CoolEpikSwagger** [comment]
Your logic is flawed lol

**Comment by CoolEpikSwagger** [comment]
> 

Uploading fan.mp4…

**Comment by CoolEpikSwagger** [comment]
https://user-images.githubusercontent.com/120762934/211455763-2bfbb747-911d-483b-8060-d1766d74418a.mp4



> trim.2A7D96A7-0B42-4071-8D96-A31DE5D0B585.mov

**Comment by sefinek24** [comment]
https://user-images.githubusercontent.com/92880265/211455824-aab21e2e-fb4e-46e6-b938-8bc96107894b.mp4

**Comment by CoolEpikSwagger** [comment]
https://user-images.githubusercontent.com/120762934/211455832-c4686aaf-8e08-4e5d-bc8d-c2defb4e1593.mp4

**Comment by CoolEpikSwagger** [comment]
![05367493d6b1471c52cb30abcec8531738b19ceecd751c13120753f780a4e216_1 jpg](https://user-images.githubusercontent.com/120762934/211455901-9ef869a2-e1c6-493e-a603-29ec111906f6.jpg)

**Comment by CoolEpikSwagger** [comment]
https://user-images.githubusercontent.com/120762934/211456001-25e5f6b2-f4af-4dcb-abf8-92a2438b0ff0.mp4

**Comment by sefinek24** [comment]
https://user-images.githubusercontent.com/92880265/211456179-e7716618-6070-467f-a591-b8d8d78f661c.mp4

**Comment by moom825** [comment]
> Your logic is flawed lol

bro...are you fucking stupid...look at the source code. look at the precompiled source code, your logic is so stupid that its not possible. the source code litterly does not allow for what your saying. it is litterly impossible, just read the src. and if you think i modified the precompiled binaries, download a tool called dnspy, and read the source code of the binaries. then accually prove to me that it is logging. oh wait, let me guess, your some skid on the internet that does not know how to read nor write code, so you obvoisly not gonna do that. ask anyone more experencied.

**Comment by CoolEpikSwagger** [comment]
> > Your logic is flawed lol
> 
> bro...are you fucking stupid...look at the source code. look at the precompiled source code, your logic is so stupid that its not possible. the source code litterly does not allow for what your saying. it is litterly impossible, just read the src. and if you think i modified the precompiled binaries, download a tool called dnspy, and read the source code of the binaries. then accually prove to me that it is logging. oh wait, let me guess, your some skid on the internet that does not know how to read nor write code, so you obvoisly not gonna do that. ask anyone more experencied.

bro can you pls fix ur errors

bro...are you fucking stupid...look at the source code. look at the precompiled source code, your logic is so stupid that its not possible. the source code literally does not allow for what your saying. it is literally impossible, just read the src. and if you think I modified the precompiled binaries, download a tool called dnspy, and read the source code of the binaries. then actually prove to me that it is logging. oh wait, let me guess, your some skid on the internet that does not know how to read nor write code, so you obviously not gonna do that. ask anyone more experienced.**

**Comment by CoolEpikSwagger** [comment]
Also its you're not your idiot

**Comment by moom825** [comment]
MB, but point still stands

**Comment by CoolEpikSwagger** [comment]
I did not read what you said, all I did was correct it, can you rephrase it without ur spelling and grammar errors, kthx.

**Comment by moom825** [comment]
tldr: the code does not allow for what your stating, which can be easily viewable and provable

**Comment by CoolEpikSwagger** [comment]
explain what provable is

**Comment by moom825** [comment]
the word? https://www.merriam-webster.com/dictionary/provable

**Comment by CoolEpikSwagger** [comment]
> the word? https://www.merriam-webster.com/dictionary/provable

elaborate on what "word" means

**Comment by moom825** [comment]
no

**Comment by CoolEpikSwagger** [comment]
ok skid

**Comment by moom825** [comment]
well, if you dont really have an compelling evidence about me logging
people, im just gonna close this issue.

**Comment by CoolEpikSwagger** [comment]
Actually I do have proof.

**Comment by moom825** [comment]
which is?

**Comment by CoolEpikSwagger** [comment]
![image](https://user-images.githubusercontent.com/120762934/211521015-99a19fda-0509-494c-a059-33abc2d80684.png)
![image](https://user-images.githubusercontent.com/120762934/211521038-3d2ebb2a-65cf-41c6-932c-894119d7704f.png)
![image](https://user-images.githubusercontent.com/120762934/211521097-09bdfbe7-abc4-4fea-83ef-501080ea00ac.png)
Ps there was no way they could have gotten without this repos since this was the only thing I used, And I didn't even share around the EXE file, Meaning one of ur dependencies are either a trojan or a stealer because this was all I used and got logged LOL

**Comment by moom825** [comment]
funny how your the only one with this issue. I think if someone somehow infected my dependencies, they would be targeting more people than just you. like I would have a bunch of people joining my server and acting the same you are and going like "you hacked me". but that has not happened, so this is what i think, you may have sent the exe and forgot about it, you have already been hacked and someones trolling you, you uploaded it to virustotal (which someone can download the exe from)

**Comment by moom825** [comment]
and also when i asked for proof i meant in code, not in speculation

**Comment by CoolEpikSwagger** [comment]
bro I have NOT sent the exe, I know that much, nor did I upload it to virustotal

**Comment by moom825** [comment]
can we move this to a discord chat because github is kinda annoying to have a convo over

**Comment by CoolEpikSwagger** [comment]
nah, i'd rather people actually see this so they dont fall for this shit

**Comment by moom825** [comment]
dont fall for what...your stupidity? read the fucking source code and tell
me where im logging people. if your next message is not that, then im just
gonna close this issue

**Comment by CoolEpikSwagger** [comment]
how abt you go read the source urself LMFAO, also if you close the issue you fuck men

**Comment by CoolEpikSwagger** [comment]
![image](https://user-images.githubusercontent.com/120762934/211523883-fe30b043-f3d8-4f8f-b48f-325d65b91af2.png)
HARAM

**Comment by CoolEpikSwagger** [comment]
> huh? that aint me?

its one of ur goons, one that create that shitty crosswood game
